fem/fespace.cpp

```diff
@@ -3249,6 +3248,7 @@ void QuadratureInterpolator::Mult(
             // Q4
             case 125125: eval_func = &Eval3D<3,125,125>; break;
             case 125216: eval_func = &Eval3D<3,125,216>; break;
+               //case 125512: eval_func = &Eval3D<3,125,512>; break;
```

Comment:
Remove

Reply:
I don't see it anymore.

Reply:
It must have been removed before the merge with master.